test/Makefile

```diff
@@ -42,8 +42,8 @@ ltp: $(LINUXKIT) test-ltp.img.tar.gz
 	$(call check_test_log, test-ltp.log)
 ### ------
 
-test: gcp-hack
+test:
 	@rtf -l build -v run -x
 
-test-pr: gcp-hack
```

Comment:
With `gcp-hack` no longer a dependency, can we get rid of the `gcp-hack` structure? It is [here](https://github.com/linuxkit/linuxkit/blob/71a95bff490e7d4505767dbf337b769c58e46f16/test/Makefile#L21-L30)

Reply:
Not quite yet. It's still being used by LinuxKitCI and deleting it now would make this PR red 😉 
If we merge this, I can come in and clean up afterwards.
Not only can `gcp-hack` disappear but so can the `ci`, `ci-pr` `ci-tag` targets in the top-level file.

I also look at moving the LTP tests to RTF outside of this PR so things will be super clean

Reply:
Ah, I should have realized that. Of course it makes sense.